src/response/file.rs

```diff
@@ -59,7 +59,7 @@ enum Open {
 struct FileStream {
     remaining: usize,
     buffer: Vec<MaybeUninit<u8>>,
-    file: Option<TokioFile>,
+    file: TokioFile,
```

Comment:
Local benchmarks show that FileStream is dropped when expected meaning that Option is unnecessary here. I'm curious how perform is across platforms though. Having to change this is rather insignificant with regards to effort. Therefore, I prefer to error on the side of caution.